src/GraphQL/GraphQLBuilderExtensions.cs

```diff
@@ -953,5 +954,101 @@ public static IGraphQLBuilder AddMetrics(this IGraphQLBuilder builder, Func<Exec
             return builder;
         }
         #endregion
+
+        #region - AddExecutionStrategySelector -
```

Comment:
I added both `AddExecutionStrategySelector` and `AddExecutionStrategy` builder methods.

Reply:
👍